RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1684,12 +1684,11 @@ private void parseOfficeTrustRecords(AbstractFile regFile, BufferedReader reader
      * @returnv BlackboardArtifact or a null value
      */
     private BlackboardArtifact createAssociatedArtifact(String filePathName, BlackboardArtifact bba) {
-        org.sleuthkit.autopsy.casemodule.services.FileManager fileManager = currentCase.getServices().getFileManager();
         String fileName = FilenameUtils.getName(filePathName);
         String filePath = FilenameUtils.getPath(filePathName);
         List<AbstractFile> sourceFiles;
         try {
-            sourceFiles = fileManager.findFiles(dataSource, fileName, filePath); //NON-NLS
+            sourceFiles = currentCase.getSleuthkitCase().getFileManager().findFilesExactNameExactPath(dataSource, fileName, filePath);
             if (!sourceFiles.isEmpty()) {
                 for (AbstractFile sourceFile : sourceFiles) {
                     if (sourceFile.getParentPath().endsWith(filePath)) {
```

Comment:
I can't tell if this is here to get exact matches or not...

Reply:
I'm pretty sure it is. I'll remove it.